packages/secret-numbers/src/schema/Account.ts

```diff
@@ -7,6 +7,9 @@ import {
   secretToEntropy,
 } from '../utils'
 
+// TODO: preferably import this type from ripple-keypairs
+// import type { Algorithm } from 'ripple-keypairs'
```

Comment:
```
Module '"ripple-keypairs"' declares 'Algorithm' locally, but it is not exported. ts(2459)
index.d.ts(2, 15): 'Algorithm' is declared here.
```
I tried to import the `Algorithm` type from `ripple-keypairs`, but I get the above error. 
I'm confused, `Algorithm` is exported here: https://github.com/XRPLF/xrpl.js/blob/be732a4a6b7bdf58d0efff084b76a66f3f625fe6/packages/ripple-keypairs/src/types.ts#L2